src/main/java/edu/ucla/library/iiif/auth/handlers/AbstractAccessTokenHandler.java

```diff
@@ -152,8 +152,7 @@ public static final void handleFailure(final RoutingContext aContext) {
         try {
             error = (ServiceException) aContext.failure();
         } catch (final ClassCastException details) {
-            aContext.fail(HTTP.INTERNAL_SERVER_ERROR, details);
-            LOGGER.error(MessageCodes.AUTH_010, details);
+            aContext.next();
```

Comment:
Does this fix the loop? Is the exception info lost here or is it retrievable in failure handling?

Reply:
It fixes the loop but it isn't a robust fix. The exception info is lost here (which isn't particularly important, since it's just communicating that this error handler isn't the right one for the error, so a different error handler should be called via `aContext.next()`) but I would like to rewrite this code so that it doesn't throw exceptions. And due to the loop, the logging statement was never being reached anyway.

Reply:
> but I would like to rewrite this code so that it doesn't throw exceptions

Okay, are you doing that on this ticket or creating a new ticket?

Reply:
will create new ticket

Reply:
SERV-349